ifrs17-template/Files/TransactionalData/Actuals_CH_2020_12.csv

```diff
@@ -1,55 +1,55 @@
-﻿@@Main,,,,
-ReportingNode,Year,Month,,
-CH,2020,12,,
-@@Actual,,,,
-DataNode,AocType,ValueType,AccidentYear,Value
-DT1.1,CF,PR,,400
-DT1.1,CF,NIC,,-280
-DT1.1,CF,ACA,,-10
-DT1.1,CF,AEA,,-5
-DT1.2,CF,NIC,2020,-280
-DT2.1,CF,PR,,400
-DT2.1,CF,NIC,,-280
-DT2.1,CF,ACA,,-10
-DT2.1,CF,AEA,,-5
-DT1.3,CF,PR,,400
-DT1.3,CF,NIC,,-280
-DT1.3,CF,ACA,,-10
-DT1.3,CF,AEA,,-5
-DT1.4,CF,PR,,400
-DT1.4,CF,NIC,,-280
-DT1.4,CF,ACA,,-10
-DT1.4,CF,AEA,,-5
-DT1.4,CF,PayablePR,,15
-DT1.4,CF,ReceivablePR,,-10
-DT1.4,CF,ReceivableNIC,,-5
-DT1.4,CF,PayableNIC,,8
-DT2.2,CF,NIC,2020,-280
-DTR1.1,CF,PR,,-200
-DTR1.1,CF,NIC,,140
-DTR1.2,CF,NIC,2020,140
-DTR2.1,CF,PR,,-200
-DTR2.1,CF,NIC,,140
-DTR2.2,CF,NIC,2020,140
-DT3.1,CF,PR,,400
-DT3.1,CF,NIC,,-280
-DT3.1,CF,ACA,,-10
-DT3.1,CF,AEA,,-5
-DT4.1,CF,PR,,400
-DT4.1,CF,NIC,,-280
-DT4.1,CF,ACA,,-10
-DT4.1,CF,AEA,,-5
-DT1.5,CF,PR,,400
-DT1.5,CF,NIC,,-280
-DT1.5,CF,ACA,,-10
-DT1.5,CF,AEA,,-5
-DT1.5,CF,ReceivablePR,,-10
```

Comment:
You should also remove these mapping keywords from the dimensions.csv, just keep it at the corresponding client projects

Reply:
ok got it. Thanks, I will now do so.

Reply:
done